speech/grpc/bin/speech-sample-async.sh

```diff
@@ -15,4 +15,4 @@
 
 SRC_DIR=$(cd "$(dirname "$0")/.."; pwd)
 java -cp ${SRC_DIR}/target/grpc-sample-1.0-jar-with-dependencies.jar \
-    com.google.cloud.speech.grpc.demos.NonStreamingRecognizeClient "$@"
+    com.google.cloud.speech.grpc.demos.AsyncRecognizeClient "$@"
```

Comment:
For the user it will become their code.  It would be good not to have code they started from marked as com.google.*,  We settled on the usage from RFC 2606 after much discussion.